src/goto-programs/interpreter.cpp

```diff
@@ -960,7 +960,8 @@ bool interpretert::unbounded_size(const typet &type)
 size_t interpretert::get_size(const typet &type)
 {
   if(unbounded_size(type))
-    return 2ULL << 32ULL;
+    return 1ULL << 31ULL;
+    //return 2ULL << 32ULL;
```

Comment:
This seems unrelated to the actual change.  It's also not clear why it's needed.